components/ListView/ListItemImages.js

```diff
@@ -43,10 +46,43 @@ class ListItemImages extends React.Component {
     this.props.setModalDeleteImageVisible(true);
   }
 
+  handleLogsShow() {
+    this.setState(prevState => ({ logsExpanded: !prevState.logsExpanded, fetchingLogs: !prevState.logsExpanded }));
+    fetchComposeLogApi(this.props.listItem.id).then(
+      logs => {
+        this.setState({ logsContent: logs, fetchingLogs: false });
+      },
+      () => {
```

Comment:
I totally hate this style, but eslint rules of this project was forcing me to have this syntax.